src/pages/member/CouponCollectionAdminPage.tsx

```diff
@@ -37,40 +37,47 @@ const CouponCollectionAdminPage: React.VFC = () => {
       key: 'expired',
       tab: formatMessage(usersMessages.tab.expired),
       coupons: coupons.filter(
-        coupon =>
-          (coupon.couponCode.couponPlan.endedAt && coupon.couponCode.couponPlan.endedAt.getTime() < Date.now()) ||
-          coupon.status.used,
+        coupon => coupon.couponCode.couponPlan.endedAt && coupon.couponCode.couponPlan.endedAt.getTime() < Date.now(),
```

Comment:
NITS: equivalent to `coupon.status.outdated` ?

Reply:
not equl

sql from voucher_status

```
SELECT
  DISTINCT voucher.id AS voucher_id,
  (
    (
      (voucher_plan.started_at IS NOT NULL)
      AND (voucher_plan.started_at > now())
    )
    OR (
      (voucher_plan.ended_at IS NOT NULL)
      AND (voucher_plan.ended_at < now())
    )
  ) AS outdated,
  (t.order_id IS NOT NULL) AS used
```